src/scss/components/bars-rulers.scss

```diff
@@ -5,23 +5,25 @@
 @use '../themes/default' as default;
 
 $path2e-bar-color: rgba(51, 51, 51, 1);
+$width-1px: 1px;
+$unset: unset;
```

Comment:
like here

Reply:
> Looks good, but i don't undersand why you replaced literals like `none` and `unset` with variables that are called the same. That just makes the code harder to read IMO.

I was playing with those a bit (because some of those might come back in 1.6, when I think we should do the layout update to try to get rid of the trait_ and rare_ extras), and left those in a force of habit. We can certainly remove them.